mappings/net/minecraft/block/state/BlockStateDataArray.mapping

```diff
@@ -0,0 +1,17 @@
+CLASS none/atr net/minecraft/block/state/BlockStateDataArray
+	FIELD a states [Lnone/asm;
+	FIELD b expander Lnone/atu;
+	FIELD c sizeBits I
+	FIELD d stateCount I
+	METHOD <init> (ILnone/atu;)V
+		ARG 0 sizeBits
+		ARG 1 expansionDelegate
+	METHOD a getSizeBytes ()I
+	METHOD a getBlockState (I)Lnone/asm;
+		ARG 0 blockStateId
+	METHOD a getBlockStateId (Lnone/asm;)I
```

Comment:
`idToBlockState` and `blockStateToId` ? `get...` sounds like a getter which this isn't?